spec/features/assigned_articles_spec.rb

```diff
@@ -21,7 +21,7 @@
     # we need to use VCR to avoid getting stopped by WebMock
     VCR.use_cassette('assigned_articles_view') do
       visit "/courses/#{course.slug}/articles/assigned"
-      page.has_content?('Nancy Tuana')
+      expect(page).to have_content('Nancy Tuana', wait:20)
```

Comment:
I'm not certain about how `page.has_content?` works, but `have_content` automatically waits and continues checking the page (up to a certain amount of time) until the expected content is found, so there's no need for the `wait` argument here.